src/Mobs/Spider.cpp

```diff
@@ -43,9 +43,15 @@ void cSpider::EventSeePlayer(cEntity * a_Entity)
 		return;
```

Comment:
QueueLightChunk should be a lightweight operation, as it only adds the chunk to the queue.